helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -392,14 +392,20 @@ alfresco-admin-app:
     pullPolicy: IfNotPresent
   applications:
     allow_custom_resources: true
+    config_ecm_host: ""
```

Comment:
why do we add `config_ecm_host`?

Seems that we need `alfresco_content_services:    enabled: false` in order to perform the `if` check?